cli/cmd/identity.go

```diff
@@ -47,16 +47,38 @@ func newCmdIdentity() *cobra.Command {
 		Use:   "identity [flags] (PODS)",
 		Short: "Display the certificate(s) of one or more selected pod(s)",
 		Long: `Display the certificate(s) of one or more selected pod(s).
-		
+
 This command initiates a port-forward to a given pod or a set of pods and fetches the TLS certificate.
 		`,
-		Example: ` 
+		Example: `
  # Get certificate from pod foo-bar in the default namespace.
  linkerd identity foo-bar
-		
+
  # Get certificate from all pods with the label name=nginx
  linkerd identity -l name=nginx
 		`,
+		ValidArgsFunction: func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
```

Comment:
This is so cool, and worked flawlessly! Especially that it takes into account the `--namespace` flag and filters the suggestions!